src/docs/product/performance/web-vitals.mdx

```diff
@@ -8,87 +8,90 @@ description: "Learn more about Web Vitals in Sentry, and how they give you a bet
+Google's "Good", "Needs Improvement", and "Poor" thresholds are used to classify data points into green, yellow, and red for the corresponding Web Vitals. "Needs improvement" is referred to as "Meh" in Sentry.
 
-#### Cumulative Layout Shift (CLS)
+| Web Vital                                                       | Good         | Meh          | Poor         |
+| --------------------------------------------------------------- | ------------ | ------------ | ------------ |
+| [Largest Contentful Paint](#largest-contentful-paint-lcp) (LCP) | <= 2.5s      | <= 4s        | > 4s         |
+| [First Input Delay](#first-input-delay-fid) (FID)               | <= 100ms     | <= 300ms     | > 300ms      |
+| [Cumulative Layout Shift](#cumulative-layout-shift-cls) (CLS)   | <= 0.1       | <= 0.25      | > 0.25       |
+| [First Paint](#first-paint-fp) (FP)                             | <= 1s        | <= 3s        | > 3s         |
+| [First Contentful Paint](#first-contentful-paint-fcp) (FCP)     | <= 1s        | <= 3s        | > 3s         |
+| [Time To First Byte](#time-to-first-byte-ttfb) (TTFB)           | <= 100ms     | <= 200ms     | > 600ms      |
 
-[Cumulative Layout Shift (CLS)](https://web.dev/cls/) is the sum of individual layout shift scores for every unexpected element shift during the rendering process. Imagine navigating to an article and trying to click a link before the page finishes loading. Before your cursor even gets there, the link may have shifted down due to an image rendering. Rather than using duration for this Web Vital, the CLS score represents the extent of the disruptive and visually unstable shifts.
+<Note>
 
-![Example of Cumulative Layout Shift](./diagram-transaction-cls.png)
+Some Web Vitals such as FP, FCP, LCP, and TTFB are measured relative to the start of the transaction. Values may differ when compared to values generated with other tools such as [Lighthouse](https://github.com/GoogleChrome/lighthouse).
 
-Each layout shift score is calculated using an impact and distance fraction. The impact fraction is the total visible area that the element affects between the two rendered frames. The distance fraction measures the distance it has moved relative to the viewport.
+</Note>
 
-```
-Layout Shift Score = Impact Fraction * Distance Fraction
-```
+## Distribution Histogram
 
-Let’s take a look at the example above which has one unstable element - the body text. The impact fraction is roughly 50% of the page and moves the body text down by 20%. The layout shift score is 0.1, the product of 0.5*0.2. Thus, CLS is 0.1.
+![Example of Web Vitals](./perf-web-vitals.png)
 
-### Other Web Vitals
+A histogram visualization showing the data distribution can help you identify and diagnose frontend performance problems by revealing anomalies.
```

Comment:
```suggestion
The Web Vitals histogram displays data distribution, and it can help you identify and diagnose frontend performance problems by revealing anomalies.
```